src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Expected/GenerateDesignerFileExpected.cs

```diff
@@ -27,8 +27,8 @@ static Resource()
 		public partial class Animator
 		{
 			
-			// aapt resource value: 0x7F070002
```

Comment:
Why did our id values change?

Reply:
Because we added two new fields. That changes the number of IDs we have and also the order in which they come.

Reply:
Looking at the change, it seems that the order the files were processed in changed. I need to look at making sure we do this in a specific order to make sure its consistent.

Reply:
I figured out a way to make the id's more consistent. They *should* match those being produced by `aapt` now.

Reply:
You can confirm this by looking at the Ids in the Rtxt file contents https://github.com/xamarin/xamarin-android/pull/3025/files#diff-8649defe4caffc0b9f604d92c1ad8808R78. These ids were generated by aapt2 on the same resource structure. I also included a test to make sure we produce the same results from the Rtxt as well.